<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelBScan</class>
 <widget class="QDockWidget" name="PanelBScan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>353</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>229</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>524287</width>
    <height>524287</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>B扫描</string>
  </property>
  <widget class="QWidget" name="contents">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,1">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitterAll">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="title1">
           <property name="text">
            <string>轮缘缺陷、传感器</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="BScanChart" name="chart1"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="title2">
           <property name="text">
            <string>轮辋轮辐分层缺陷</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="BScanChart" name="chart2"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="layoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="title3">
           <property name="text">
            <string>轮辋轮辐正向裂纹</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="BScanChart" name="chart3"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="title4">
           <property name="text">
            <string>轮辋轮辐负向裂纹</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="BScanChart" name="chart4"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="feature"/>
      </item>
      <item>
       <widget class="QCheckBox" name="adaptRange">
        <property name="text">
         <string>Y范围自适应</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="timelineEnable">
        <property name="text">
         <string>时间轴</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="timelineContainer" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSlider" name="timelinePos">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timelineText">
           <property name="text">
            <string>#</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="timelineRange"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BScanChart</class>
   <extends>QGraphicsView</extends>
   <header>Plot/BScanChart.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
